build/tasks/node_smoke_tests.js

```diff
@@ -19,26 +17,34 @@ if ( !verifyNodeVersion() ) {
 // important so that the tests & the main process don't interfere with
 // each other, e.g. so that they don't share the `require` cache.
 
-async function runTests() {
-	const promises = args.map( async( arg ) => {
-		const [ moduleType, module ] = arg.split( ":" );
-		if ( !allowedModules.includes( moduleType ) ) {
-			throw new Error(
-				`Usage: \`node_smoke_tests [${allowedModules.join( "|" )}]:JQUERY\``
-			);
-		}
-		const dir = `./test/node_smoke_tests/${moduleType}`;
-		const files = await fs.promises.readdir( dir, { withFileTypes: true } );
-		const testFiles = files.filter( ( testFilePath ) => testFilePath.isFile() );
-		return Promise.all(
-			testFiles.map( ( testFile ) =>
-				exec( `node "${dir}/${testFile.name}" "${module}"` )
-			)
+async function runTests( sourceType, module ) {
+	if ( !allowedModules.includes( sourceType ) ) {
+		throw new Error(
+			`Usage: \`node_smoke_tests [${allowedModules.join( "|" )}]:JQUERY\``
 		);
-	} );
-	Promise.all( promises ).then( () => {
-		console.log( `Node smoke tests passed for ${args.join( ", " )}.` );
-	} );
+	}
+	const dir = `./test/node_smoke_tests/${sourceType}`;
+	const files = await fs.promises.readdir( dir, { withFileTypes: true } );
+	const testFiles = files.filter( ( testFilePath ) => testFilePath.isFile() );
+	await Promise.all(
+		testFiles.map( ( testFile ) =>
+			exec( `node "${dir}/${testFile.name}" "${module}"` )
+		)
+	);
+	console.log( `Node smoke tests passed for ${sourceType} "${module}".` );
 }
 
-runTests();
+async function runDefaultTests() {
+	await Promise.all( [
+		runTests( "commonjs", "jquery" ),
+		runTests( "commonjs", "jquery/slim" ),
+		runTests( "commonjs", "./dist/jquery.js" ),
+		runTests( "commonjs", "./dist/jquery.slim.js" ),
+		runTests( "module", "jquery" ),
+		runTests( "module", "jquery/slim" ),
+		runTests( "module", "./dist-module/jquery.module.js" ),
+		runTests( "module", "./dist-module/jquery.slim.module.js" )
+	] );
+}
```

Comment:
I like it. It reduces the noise in `package.json` and those tests are quick anwyay.